packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1801,22 +1799,26 @@ export class ContainerRuntime
 		const snapshotSeqNumber = snapshot.sequenceNumber;
 		assert(snapshotSeqNumber !== undefined, "snapshotSeqNumber should be present");
 
+		// This assert fires if we get a snapshot older than the snapshot we loaded from. This is a service issue.
```

Comment:
I'd add a bit more details on why, i.e.: snapshots should move only forward.
If we observe that we got older snapshot, then likely file has been overwritten or service lost data.
From Loop perspective this is data loss scenario, so maybe we should throw appropriate exception (with all the properties that the rest of the system expects) instead of assert?